### PR TITLE
Add !downorjustme command

### PR DESCRIPTION
**Pull request by tabjy**
This pull request added the `!downorjustme <hostname|url>` command that quickly checks whether a web service or a host is up and running. 

Examples:
 - `!downorjustme google.com`
 - `!downorjustme 8.8.8.8`
 - `!downorjustme https://google.com`
 - `!downorjustme https://google.com/404`

By default, the following destinations are disallowed due to security concerns:
 - a loopback address
 - a multicase address
 - an address in subnet `10.0.0.0/8`

More blacklisted subnets and IP addresses (eg. `1.1.1.1/32`) can be added as necessary.

Tests included. No assembly required.

### CODE REVIEW

File: vilebot/src/main/java/com/oldterns/vilebot/Vilebot.java [hunk 1]
```diff
@@ -117,7 +118,7 @@ public static void main( String[] args )
...
...uit() ).addListener( new AnswerQuestion() ).addListener( new Ascii() ).addListener( new ChatLogger() ).addListener( new Church() ).addListener( new Countdown() ).addListener( new Decide() ).addListener( new Excuses() ).addListener( new FakeNews() ).addListener( new Fortune() ).addListener( new GetInfoOn() ).addListener( new Help() ).addListener( new ImageToAscii() ).addListener( new Inspiration() ).addListener( new Jaziz() ).addListener( new Jokes() ).addListener( new Kaomoji() ).addListener( new Karma() ).addListener( new KarmaRoll() ).addListener( new KarmaTransfer() ).addListener( new LastMessageSed() ).addListener( new LastSeen() ).addListener( new Markov() ).addListener( new News() ).addListener( new Omgword() ).addListener( new Ops() ).addListener( new QuotesAndFacts() ).addListener( new RemindMe() ).addListener( new RockPaperScissors() ).addListener( new Summon() ).addListener( new Trivia() ).addListener( new Ttc() ).addListener( new TwitterCorrection() ).addListener( new UrlTitleAnnouncer() ).addListener( new UrlTweetAnnouncer() ).addListener( new Userlists() ).addListener( new UserPing() ).addListener( new Weather() ).addListener( new DownOrJustMe() ).buildConfiguration();
```
Comment (cuijulian) [inline vilebot/src/main/java/com/oldterns/vilebot/Vilebot.java]: Listener `DownOrJustMe()` should be added in alphabetical order with the other listeners (so between `new Auth()` and `new GetLog()`.

File: vilebot/src/test/java/vilebot/handlers/user/DownOrJustMeTest.java [hunk 2]
```diff
@@ -0,0 +1,110 @@
+package vilebot.handlers.user;
+
+import com.oldterns.vilebot.handlers.user.DownOrJustMe;
+import org.junit.Before;
+import org.junit.Test;
+import org.pircbotx.hooks.events.MessageEvent;
+
+import static org.mockito.Mockito.*;
```
Comment (cuijulian) [inline vilebot/src/test/java/vilebot/handlers/user/DownOrJustMeTest.java]: Not a blocker, but wildcard imports are not recommended.

File: vilebot/src/main/java/com/oldterns/vilebot/Vilebot.java (hunk 1 above)
Comment (tabjy) [reply vilebot/src/main/java/com/oldterns/vilebot/Vilebot.java]: Out of curiosity: is there a reason behind writing everything on one single line or it's simply a dogma of the church?

Comment (cuijulian) [reply vilebot/src/main/java/com/oldterns/vilebot/Vilebot.java]: Yes. We'll go with dogma of the church :+1:

Comment (tabjy) [reply vilebot/src/main/java/com/oldterns/vilebot/Vilebot.java]: Fixed in 863a74f974bfeaf6799ee12a73f205f349259d7d.

File: vilebot/src/test/java/vilebot/handlers/user/DownOrJustMeTest.java (hunk 2 above)
Comment (tabjy) [reply vilebot/src/test/java/vilebot/handlers/user/DownOrJustMeTest.java]: Fixed in 863a74f974bfeaf6799ee12a73f205f349259d7d.

**Pull request merged**